locksmith/src/operations/wedlocksOperations.ts

```diff
@@ -192,8 +195,10 @@ const getAttachments = async ({
     attachments.push({ path: dataURI })
   }
 
+  const { isEvent } = types ?? {}
+
   // Add ICS attachment when event is present
-  if (event) {
+  if (isEvent && event) {
```

Comment:
add `event` attachment only when lock is eventType

Reply:
Great!